<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1604</width>
    <height>984</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Video">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>155</y>
      <width>790</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border:1px solid black;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Total_Time">
    <property name="geometry">
     <rect>
      <x>938</x>
      <y>611</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Duration">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>616</y>
      <width>661</width>
      <height>18</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_Time">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>611</y>
      <width>82</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_songTitle">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>360</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name_File</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>781</x>
      <y>661</y>
      <width>231</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Volume">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_Volume">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>650</y>
      <width>491</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Backward">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Play_Pause">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Stop">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Seek_Forward">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>790</y>
      <width>351</width>
      <height>102</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_Previous">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Next">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    border-radius: 50px;
    background-color: rgb(255, 255, 255);
    color: rgb(52, 51, 64);
    min-width: 100px;
    max-width: 100px;
    min-height: 100px;
    max-height: 100px;
    padding: 0px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1604</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpenFile">
    <property name="title">
     <string>OpenFile</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <addaction name="menuOpenFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
